package.json

```diff
@@ -32,15 +32,18 @@
   "deprecated": false,
   "description": "TODO",
   "devDependencies": {
+    "@types/jest": "^26.0.23",
+    "@types/react": "^17.0.4",
+    "@types/react-native": "^0.64.4",
+    "@types/react-test-renderer": "^17.0.1",
     "react": "^16.9.0",
-    "react-native": "^0.61.5"
+    "react-native": "^0.61.5",
+    "typescript": "^4.2.4"
```

Comment:
Maybe will remove "^" for more stability?

Reply:
Done